index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portflio</title>
+    <link rel="stylesheet" href="nina.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Festive&family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <div class="container">
+        <div class="header hoverable">
+            <h1>My Logo</h1>
+            <img src="/photos/Enabled.png" alt="Hamburger Icon">
+        </div>
+    
+    
+        <div class="Main">
+            <div class="title hoverable">
+                <h1>I’m Mariane Glad to see you!</h1>
+            </div>
+            <div class="text hoverable">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="social hoverable">
+                <h4 class="hoverable">LET’S CONNECT</h4>
+                <ul class="media">
+                    <li><img class="twi" src="/photos/twitter.jpg" alt="Twitter Icon"></li>
+                    <li><img class="lin" src="/photos/Linkedin-icon.jpg" alt="Linkedin Icon"></li>
+                    <li><img class="mil" src="/photos/medium.jpg" alt="Mil Icon"></li>
+                    <li><img class="git" src="/photos/github.jpg" alt="github Icon"></li>
+                    <li><img class="angel" src="/photos/angellist.jpg" alt="Angellist Icon"></li>
+                    
+                </ul>
+            </div>
```

Comment:
- [ ] For semantic reasons Please wrap this whole section with `<section>` tag instead of `<div>` tag.
- [ ] nice work adding the social media icons inside an unordered list but kindly wrap each `<img>` tag with `<a>` tag and put your social media links because at the end this will be your portfolio.
for example, make it like that:
`<li><a href="your link"><img class="twi" src="/photos/twitter.jpg" alt="Twitter Icon"></a></li>`